src/client/index.ts

```diff
@@ -163,7 +173,25 @@ export class Resend {
   }
 
   /**
-   * Sends an email.
+   * Sends an email by providing an object with all the email options. This is the recommended approach.
```

Comment:
```suggestion
   * Sends an email
```
they won't have context about the alternative so referring to object & recommended could be confusing